Util/DomainWhitelist.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\Util;
+
+/**
+ * @final
+ */
+class DomainWhitelist
```

Comment:
You may add the final keyword too.

Reply:
unfortunately I can't because I'm mocking this class in `RedirectToServiceControllerTest.php`